site/docs/encoding/scale.md

```diff
@@ -321,6 +321,38 @@ For example, the following plot has a binned field on the `color` channel.
 
 Similar to [ordinal](#ordinal) color scales, a custom [`range`](#range) or [`scheme`](#scheme) can be specified for binned ordinal scales.
 
+{:#quantile}
+### Quantile Scales
+
+Quantile scales (`"quantile"`) map a sample of input domain values to a discrete range based on computed [quantile](https://en.wikipedia.org/wiki/Quantile) boundaries.
+The domain is considered continuous and thus the scale will accept any reasonable input value; however, the domain is specified as a discrete set of sample values.
+The number of values in (i.e., the cardinality of) the output range determines the number of quantiles that will be computed from the domain. To compute the quantiles, the domain is sorted, and treated as a population of discrete values.
+The resulting quantile boundaries segment the domain into groups with roughly equals numbers of sample points per group. The `range` property, if not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.
+
+Quantile scales are particularly useful for creating color or size encodings with a fixed number of output values. Using a discrete set of encoding levels (typically between 5-9 colors or sizes) sometimes supports more accurate perceptual comparison than a continuous range.
+For related functionality see [quantize scales](https://vega.github.io/vega-lite/docs/scale.html#quantize), which partition the domain into uniform domain extents, rather than groups with equal element counts.
+Quantile scales have the benefit of evenly distributing data points to encoded values. In contrast, quantize scales uniformly segment the input domain and provide no guarantee on how data points will be distributed among the output visual values.
+
+<span class="vl-example" data-name="bar_scale_quantile"></span>
+
+{:#quantize}
+### Quantize Scales
+
+Quantize scales (`"quantize"`) are similar to [linear scales](https://vega.github.io/vega-lite/docs/scale.html#linear), except they use a discrete rather than continuous range. The continuous input domain is divided into uniform segments based on the number of values in (i.e., the cardinality of) the output range.
+Each range value y can be expressed as a quantized linear function of the domain value x: _y_ = _m round(x) + b_. The `range` property, if not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.
```

Comment:
1) Since quantize supports `nice` and `zero`, add a table for it too?

2)

> Each range value y can be expressed as a quantized linear function of the domain value x: _y_ = _m round(x) + b_. 
> The `range` property, if not specified, is default to 4 uniform segments between minimum and maximum range value determined by Vega-lite based on mark types.

This reads a bit confusing.  If I understand correctly, the range value here is like the first step of the scale. (A quantize scale maps continuous to quantized values, then maps the quantized values to the specified range.)


3) 

>  default to 4 uniform segments between minimum and maximum range value determined 

If there are 4 points, there are 3 uniform segments. 


4)

> determined by Vega-lite based on mark types.

Readers will be confused "how is it based on mark types"?  (Even I don't know what this means.

Reply:
3) Actually, there are 4 segments for 4 points (example: https://github.com/vega/vega/blob/22cd624cec5815d1263ca6c1e8effe58cee4f2d1/spec/scales-discretize.vg.json)

4) I meant the minimum and maximum range is determined based on type, but this seems confusing so I'll remove it.